types/jquery/index.d.ts

```diff
@@ -61847,63 +61876,108 @@ jQuery.easing.easeInCubic = function ( p ) {
 </head>
 <body>
 ​
-<button value="Test" name="Test" id="checkMetaKey">Click me!</button>
-<div id="display"></div>
+<div id="log"></div>
 ​
 <script>
-$( "#checkMetaKey" ).click(function( event ) {
-  $( "#display" ).text( event.metaKey );
+$( document ).on( "mousemove", function( event ) {
+  $( "#log" ).text( "pageX: " + event.pageX + ", pageY: " + event.pageY );
 });
 </script>
 </body>
 </html>
 ```
          */
-        metaKey: boolean;
+        pageX: number;
         /**
-         * The namespace specified when the event was triggered.
+         * The mouse position relative to the top edge of the document.
          *
-         * @see \`{@link https://api.jquery.com/event.namespace/ }\`
-         * @since 1.4.3
-         * @example ​ ````Determine the event namespace used.
+         * @see \`{@link https://api.jquery.com/event.pageY/ }\`
+         * @since 1.0.4
+         * @example ​ ````Show the mouse position relative to the left and top edges of the document (within this iframe).
 ```html
 <!doctype html>
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>event.namespace demo</title>
+  <title>event.pageY demo</title>
+  <style>
+  body {
+    background-color: #eef;
+  }
+  div {
+    padding: 20px;
+  }
+  </style>
   <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
 </head>
 <body>
 ​
-<button>display event.namespace</button>
-<p></p>
+<div id="log"></div>
 ​
 <script>
-$( "p" ).on( "test.something", function( event ) {
-  alert( event.namespace );
-});
-$( "button" ).click(function( event ) {
-  $( "p" ).trigger( "test.something" );
+$( document ).on( "mousemove", function( event ) {
+  $( "#log" ).text( "pageX: " + event.pageX + ", pageY: " + event.pageY );
 });
 </script>
 </body>
 </html>
 ```
          */
-        namespace: string;
+        pageY: number;
+        screenX?: number;
+        screenY?: number;
+        /** @deprecated */
+        toElement?: Element;
+
+        // #endregion
+
+        // region PointerEvent
```

Comment:
Any reason you open the region twice? Is there some way our tooling isn't working well for you here?

Reply:
WebStorm uses `// #region` to designate a region start while VS Code uses `// region`. They can both (must, in this case) be terminated using `// #endregion`. Using both `// #endregion` and `// endregion` results in odd behavior in both editors.

Reply:
I wonder if we could support `// #region` as well for compatibility. @uniqueiniquity

Reply:
@andy-ms We do use `// #region`, not `// region`. Not sure why VS code would be picking up the latter and not the former.

Reply:
See [here](https://github.com/Microsoft/TypeScript/blob/a4a5b3806e92ddced9f82b2c341e78bd91c041e4/src/services/outliningElementsCollector.ts#L86) for specifics.

Reply:
@leonard-thieu I can confirm that it's working in vscode with only `// #region` comments, no `// region`.

Reply:
Sorry, I had it reversed. WebStorm uses `// region`.

#### WebStorm using `// region`
![image](https://user-images.githubusercontent.com/5265970/46624576-2caff380-caff-11e8-91d3-df4d191febca.png)

#### WebStorm using `// #region`
![image](https://user-images.githubusercontent.com/5265970/46624587-35a0c500-caff-11e8-8488-75fac9eab0e0.png)

Reply:
Maybe we should add support in typescript without the `#` then.